syntax = "proto3";
package WeatherData;

option csharp_namespace = "unwdmi.Protobuf";

message Measurement {
	uint32 StationID = 1;
	int64 DateTime = 2;
	double Humidity = 3;
	float WindSpeed = 4;
	float CloudCover = 5;
}

message WeatherStation {
	uint32 StationNumber = 1;
	string Name = 2;
	string Country = 3;
	double Latitude = 4;
	double Longitude = 5;
	double Elevation = 6;
	bool IgnoreStation = 7;
}

message Country {
	string Country = 1;
	bool Ignore = 2;
}